zubhub_frontend/zubhub/src/views/pageWrapperScripts.js

```diff
@@ -43,7 +43,7 @@ export const handleScrollTopClick = (e, ref) => {
  * @todo - describe function's signature
  */
 export const handleProfileMenuOpen = e => {
-  return { anchor_el: e.currentTarget };
+  return { anchor_el: e.currentTarget , displayMenuOpen:'block'};
```

Comment:
I looked at your changes and I can't see where these were `displayMenuOpen` was used. perhaps remove it?

Reply:
> @souad988 I should also mention that there are a couple of places in the frontend that we make use of the
> 
> component and I'm pretty sure that we are not using `disableScrollLock={true}`. Maybe you can add this prop to all the places?
> most of the places don't require the menu to close when scrolling is attempted so they should be fine without `useScrollTrigger`. only do the `useScrollTrigger` thing for the locale selector and the profile menu (like you already did)

the other component are Select ones not Menu and Select doesn't have disableScrollLock attribute i tried using `<Select  MenuProps={{ disableScrollLock=true}} >` since it has MenuItems as children, but didn't work, i'm new to material ui  i would like to learn, if you can give me some hints i appreciate

Reply:
> Yes I forgot to mention, if you notice that your development environment added dirty diffs to the package-lock.json file, you should run `git checkout package-lock.json` on the frontend directory to discard that change before staging and committing. Right now your package-lock.json file contains dirty diffs we wouldn't be happy to merge into master.

Done